common/src/main/resources/tachyon-default.properties

```diff
@@ -13,6 +13,7 @@
 
 # Default configuration properties for Tachyon
 # All the properties except Build properties are override-able via tachyon-site.properties file.
+# Setting a property to -1 denotes that this property is not set but will use system default value.
```

Comment:
Is this universally true? I mean, if any property is set to -1, TachyonConf will take care of it somehow? Or, do only some of the properties have this ability to set to "-1"?